common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/DiagnosisKey.java

```diff
@@ -34,13 +34,20 @@
    * reject any diagnosis keys that do not have a rolling period of a certain fixed value. See
    * https://developer.apple.com/documentation/exposurenotification/setting_up_an_exposure_notification_server
    */
+  public static final int KEY_DATA_LENGTH = 16;
   public static final int MIN_ROLLING_PERIOD = 0;
   public static final int MAX_ROLLING_PERIOD = 144;
+  public static final int MIN_DAYS_SINCE_ONSET_OF_SYMPTOMS = -14;
+  public static final int MAX_DAYS_SINCE_ONSET_OF_SYMPTOMS = 4000;
+  public static final int MIN_TRANSMISSION_RISK_LEVEL = 1;
```

Comment:
@mibrasap why is this now 1? can we refer to any documentation?

Reply:
@hilmarf  It should be 1. It was a misunderstanding from my side in the past weeks that the TRL range is 0-8.  I changed it to 0 initially,, but have discussed with Michael that it should be reverted  back to 1